examples/ble-gatt-server/src/main/java/no/nordicsemi/android/ble/ble_gatt_server/GattService.kt

```diff
@@ -191,8 +191,7 @@ class GattService : Service() {
             log(Log.DEBUG, "Device connected ${device.address}")
 
             val clientConnection = ServerConnection()
-            clientConnection.useServer(this@ServerManager)
-            clientConnection.connect(device).enqueue()
```

Comment:
The change here reads more intuitively to me i.e. why should a peripheral *have* to connect? I don't think it should.